### ISSUE TITLE
Plans?

### ISSUE THREAD

**Issue by DualBrain** [andymule/qb64-vscode#1]
Stumbled across this repo and was wondering if you've made any progress on this (possibly offline) and just haven't committed.

Thanks.

**Comment by Kroc** [comment]
No, but yes it does need an update. I didn't think anybody was using it. Is there anything you need in particular?
